coveo-functools/README.md

```diff
@@ -0,0 +1,149 @@
+# coveo-functools
+
+Introspection, finalizers, delegates, dispatchers, waiters...
+These utilities aim at increasing productivity.
+
+
+## annotation
+
+Introspect classes and callables at runtime.
+
+Can convert string annotations into their actual type reference.
+
+
+## casing / flexcase
+
+flexcase takes a "dirty" input and maps it to a python construct.
+
+The principal use case is to allow seamless translation between snake_case and camelCase and generate PEP8-compliant code over APIs that support a different casing scheme.
+
+- It introspects a function to obtain the expected argument names
+- It inspects the provided input to find matching candidates
+- It calls the function with the cleaned arguments
+
+It can also be used to allow for a certain degree of "personalization" in typically strict contexts such as configuration files and APIs. 

```

Comment:
```suggestion
It can also be used to allow for a certain degree of customization in typically strict contexts such as configuration files and APIs. 
```
Pas besoin de quotes?

Reply:
C'est personalization dans le sens que le user à l'autre bout peut décider s'il écrit ses affaires en camel, snake ou dash-snake. Ou bien, il n'a pas à se rappeller si ton app prends des dash ou des underscores. Y'a p-e moyen de rephraser?

Reply:
Ah ok. Ben personalization fonctionne, mais pas de quotes nécessaire je dirais.